silveroak-opentitan/aes/Acorn/.gitignore

```diff
@@ -0,0 +1 @@
+!AESSV.hs
```

Comment:
It gets kind of messy to maintain output files next to these for this, and some other, reasons. Any thought we could move all output to a dedicated output directory? (this would also simplify 'make clean').

Reply:
Yes, agreed, I think I would definitely like to find a better way to do this.
Several times I've been bitten by having a file I worked on for ages getting deleted by the git-remove :-(

Reply:
I also agree, Coq extraction is the difficult bit at the moment- not sure how to stop it dumping output files in the source location